src/data/content/quality.ts

```diff
@@ -5,9 +5,193 @@ const techContent = [
+            description:
+              "Selenium supports multiple browsers like chrome, firefox, safari, IE, Edge.",
+          },
+        ],
+      },
+      {
+        name: "Basic Steps in a Selenium WebDriver Script",
+        intro: "",
+        data: [
+          {
+            name: "",
+            description: "1. Create a WebDriver instance.",
+          },
+          {
+            name: "",
+            description: "2. Navigate to a webpage.",
+          },
+          {
+            name: "",
+            description: `3. Locate a web element on the webpage via <a href="https://www.selenium.dev/documentation/webdriver/elements/finders/">locators in selenium</a>. `,
+          },
+          {
+            name: "",
+            description: "4. Perform one or more user actions on the element. ",
+          },
+          {
+            name: "",
+            description:
+              "5. Preload the expected output/browser response to the action.",
+          },
+          {
+            name: "",
+            description: "6. Run test.",
+          },
+          {
+            name: "",
+            description:
+              "7. Record results and compare results from them to the expected output.",
+          },
+        ],
+      },
+      {
+        name: "Benefits of Selenium WebDriver",
+        intro: "",
+        data: [
+          {
+            name: "",
+            description: `<ul><li>It is one of the most popular Open-Source tools and is easy to get started with for testing web-based applications. It also allows you to perform <a href="https://www.selenium.dev/documentation/webdriver/getting_started/install_drivers/">cross browsers testing</a>.</li></ul> `,
+          },
+          {
+            name: "",
+            description:
+              "<ul><li>Supports multiple operating systems like Windows, Mac, Linux, Unix, etc.</li></ul>",
+          },
+          {
+            name: "",
+            description: `<ul><li>It provides compatibility with a range of languages including Python, Java, JavaScript, Ruby, etc.</li></ul>`,
+          },
+          {
+            name: "",
+            description:
+              "<ul><li>Provides support for modern browsers like Chrome, Firefox, Edge, Opera, Safari, and Internet Explorer.</li></ul>",
+          },
+          {
+            name: "",
+            description:
+              "<ul><li>Selenium WebDriver completes the execution of test scripts faster when compared to other tools</li></ul>",
+          },
+          {
+            name: "",
+            description:
+              "<ul><li>More Concise API (Application Programming interface) than Selenium RC’s</li></ul>",
+          },
+          {
+            name: "",
+            description:
+              "<ul><li>It also provides compatibility with iPhoneDriver, HtmlUnitDriver, and AndroidDriver</li></ul>",
+          },
+        ],
+      },
+      {
+        name: "Web Element",
```

Comment:
The titles are repeated twice.
![Uploading Screenshot 2022-06-01 at 16.16.41.png…]()